README.md

```diff
@@ -176,9 +176,34 @@ To support [federated tracing](https://www.apollographql.com/docs/apollo-server/
     end
     ```
 
+## Exporting the Federated SDL
+
+When using tools like [rover](https://www.apollographql.com/docs/rover/) for schema validation, etc., add a Rake task that prints the Federated SDL to a file: 
+
+```rb
+namespace :graphql do
+  namespace :federation do
+    task :dump do
+      File.open "schema.graphql", "w+" do |f|
+        f << MySchema.federation_sdl
+      end
+    end
+  end
+end
+```
+
+(This task mirrors the [`graphq:schema:dump` task](https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/rake_task.rb) included in graphql-ruby.)
```

Comment:
I think I get what you mean but this might be a little confusing without context. The example rake task has no direct relation to GraphQL's schema dump task. When I click into the Github link, there's so much more going on there. Would suggest: 

1. Removing it, I think what you explained on top about using Rover and the schema output is concise  :+1: 
2. Maybe state that there is a similar task in GraphQL Ruby without mirroring.